bot/seasons/easter/april_fools_vids.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class AprilFoolVideos(commands.Cog):
+    """A cog for april fools that gets a random april fools video from youtube."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.yt_vids = self.load_json()
+        self.youtubers = ['google']  # will add more in future
+
+    @staticmethod
+    def load_json():
+        """A function to load json data."""
+        p = Path('bot/resources/april_fools_vids.json')
```

Comment:
This path is incorrect. It should be either:

- `Path('bot/resources/easter/april_fools_vids.json')`
- `Path('bot', 'resources', 'easter', 'april_fools_vids.json')`

Reply:
oh shoot,my bad, i was testing it my own discord bot,lol